.scalafmt.conf

```diff
@@ -0,0 +1 @@
+// default settings
```

Comment:
Okay, someone has to start the bikeshedding, so here goes:

I'd like to see
```
align=none
maxColumn = 120
```

The rest of the default settings I think are good :)